updatecli/updatecli.d/headers.yaml

```diff
@@ -0,0 +1,62 @@
+name: Add header to the CLI reference documentation
+#{{ if .pipelineid }}
+pipelineid: '{{ .pipelineid }}'
+#{{ end }}
+
+scms:
+  target:
+    kind: github
+    spec:
+      owner: {{ requiredEnv .scms.owner }}
+      repository: {{ .scms.repository }}
+      branch: {{ .scms.branch }}
+      user: {{ requiredEnv .scms.owner }}
+      username: {{ .scms.username }}
+      token: {{ requiredEnv .scms.token }}
+
+sources:
+# {{ range $index, $file := .files }}
```

Comment:
I'm confused about this line and other ones, are these comments or did you mean to have something like that?

```
#{{ range $index, $file := .files }}
```

Reply:
I like this approach because using the comment allow us to use go template, while keeping the advantage of having a compliant yaml file.

Differently said:
```
{{ range $index, $file := .files }}
```
Would also work, but your IDE would complain that it's not a valid yaml file and the Updatecli jsonschema wouldn't be loaded loosing Updatecli doc and auto-completion from IDE supporting jsonschema store